udata/core/dataset/forms.py

```diff
@@ -22,27 +22,6 @@ class ChecksumForm(ModelForm):
     value = fields.StringField()
 
 
-class ChecksumWidget(object):
```

Comment:
We don't have front forms any more so all widgets are no longer used (except for the coming login/register)